pkg/process/checks/net.go

```diff
@@ -448,14 +448,17 @@ func groupSize(total, maxBatchSize int) int32 {
 }
 
 // converts the tags based on the tagOffsets for encoding. It also enriches it with service context if any
-func convertAndEnrichWithServiceCtx(tags []string, tagOffsets []uint32, serviceCtx string) []string {
-	tagCount := len(tagOffsets) + len(serviceCtx)
+func convertAndEnrichWithServiceCtx(tags []string, tagOffsets []uint32, serviceCtxs []string) []string {
```

Comment:
consider using the following, it will allow you to pass a single string to multiple
```suggestion
func convertAndEnrichWithServiceCtx(tags []string, tagOffsets []uint32, serviceCtxs ...string) []string {
```

Reply:
Good suggestion! Added